class LoginError(Exception):
    pass

class CredentialsError(Exception):
    pass